sanityv3/schemas/editors/titleEditorContentType.tsx

```diff
@@ -1,14 +1,43 @@
 import { SuperScriptRenderer, SubScriptRenderer, StrikeThroughRenderer } from '../components'
-import { IconSuperScript, IconSubScript } from '../../icons'
+import { IconSuperScript, IconSubScript, EdsBlockEditorIcon } from '../../icons'
 import { StrikethroughIcon } from '@sanity/icons'
 import type { BlockFieldType } from '../../types/schemaTypes'
+import { format_color_text } from '@equinor/eds-icons'
+
+export type TitleContentProps = {
+  highlight?: boolean
+  extraLarge?: boolean
+  isBigTitle?: boolean
+  large?: {
+    title: string
+    value: 'normal'
+    component?: ({ children }: { children: React.ReactNode }) => JSX.Element
+  }
+}
 
 // TODO: Add relevant styles for titles (i.e. highlighted text)
-export const configureTitleBlockContent = (): BlockFieldType => {
-  // const { strong, emphasis, strikethrough } = options
-  return {
+export const configureTitleBlockContent = (options: TitleContentProps = {}): BlockFieldType => {
+  const {
+    large = {
+      title: 'Large',
+      value: 'normal',
+    },
+    extraLarge = false,
+    highlight = false,
+    isBigTitle = false,
+  } = options
+
+  const largeTitleText = {
+    title: 'Large',
+    value: 'normal',
+    component: ({ children }: { children: React.ReactNode }) => <span style={{ fontSize: '42px' }}>{children}</span>,
+  }
+
+  const titleStyle = extraLarge || isBigTitle ? largeTitleText : large
+
+  const config: BlockFieldType = {
```

Comment:
this type was created by me when sanity didn't have type support for this, now looks like it has. so if you want to do a small refactoring that would be nice

- `BlockDefinition` type